components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesHistoryStorageTest.kt

```diff
@@ -760,9 +760,9 @@ class PlacesHistoryStorageTest {
         assertEquals("test error", error.exception.message)
     }
 
-    @Test(expected = InternalPanic::class)
+    @Test(expected = InternalException::class)
```

Comment:
In `PlacesError` there are two variants that look like they should not be caught by consumers - `UnexpectedPlacesException`  and `InternalPanic`.

What is `InternalException` and how does it relate to these?